src/styles/star.scss

```
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+  }
+
+  .stars--0 > &__star:nth-child(-n + 0) {
```

```suggestion
  .stars--0 > &__star:nth-child(-n + 0),
  .stars--1 > &__star:nth-child(-n + 1),
  ... {
```
Consider using commas to group selectors and avoid repeating code